os/kernel/init/os_bringup.c

```diff
@@ -284,6 +256,15 @@ static inline void os_do_appstart(void)
 
 	svdbg("Starting application init thread\n");
 
+#ifdef CONFIG_BINARY_MANAGER
+#define BINARY_MANAGER_STACKSIZE 2048
+#define BINARY_MANAGER_PRIORITY 225
+
+	pid = kernel_thread("binary_manager", BINARY_MANAGER_PRIORITY, BINARY_MANAGER_STACKSIZE, binary_manager, NULL);
+
+	if (pid < 0) {
+		sdbg("Failed to start binary manager");
+	}
 
 #ifdef CONFIG_SYSTEM_PREAPP_INIT
```

Comment:
Later you should merge this into binary manger to start from there.

Reply:
yes. I know. I will do that later